src/repositories/hive/hive.repository.mock.ts

```diff
@@ -76,7 +76,10 @@ export class MockHiveRepository extends HiveRepository {
     };
   }
 
-  verifyHiveMessage(message: Buffer, signature: string, account: ExtendedAccount): boolean {
-    return true;
+  async getAccount(author: string) {
+    const [hiveAccount] = await this._hive.database.getAccounts([author]);
+    if (process.env.TEST_PUBLIC_KEY)
+      hiveAccount.posting.key_auths.push([process.env.TEST_PUBLIC_KEY, 0]);
```

Comment:
Shouldn't the weight be whatever the required weight is for the account and not `0`? Do we check the weights when verifying signatures?